### DISCUSSION TITLE
Não consigo abrir o github/elidianaandrade/

### DISCUSSION THREAD

**Discussion by mfc75** [digitalinnovationone/dio-lab-open-source#41025] [Perguntas e Respostas] [answered]
Eu tinha criado o clone a partir de alguem que tinha criado o clone a partir da elidianaandrade/dio-lab-open-source e agora agora não consigo mais abrir o link elidianaandrade/dio-lab-open-source, quando digito este endereço, abre o endereço digitalinnovationone/dio-lab-open-source, o qual não consigo clonar

**Comment by elidianaandrade** [comment, marked as answer]
Olá, @mfc75! O repositório foi transferido da minha conta para a da DIO, agora é esse **digitalinnovationone/dio-lab-open-source** mesmo. Basta abrir o Git Bash e dar um `git clone https://github.com/digitalinnovationone/dio-lab-open-source` 😊
